From 5763aa23b9450f77a843a45d3631fff2ba82822f Mon Sep 17 00:00:00 2001
From: "DIR\\gabriel.de.s.barbosa" <gabriel.de.s.barbosa@accenture.com>
Date: Wed, 28 Feb 2018 13:44:19 -0300
Subject: [PATCH] no message

---
 receive.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/receive.js b/receive.js
index fa4acdd..e691a93 100644
--- a/receive.js
+++ b/receive.js
@@ -17,7 +17,7 @@ amqp.connect('amqp://localhost', function(err, conn) {
       //Verifica quantos pontos tem a imagem
       var segs = msg.content.toString().split('.').lenght - 1;
       //Mostra a imagem recebida
-      console.log(" [x] Received %s", msg.content.toString());
+      console.log(" [x] Received %s with %d secs", msg.content.toString(),segs);
       //Espera o tempo que ela precisa pra "trabalhar"
       setTimeout(function(){
         console.log(" [X] Done");
-- 
2.15.1.windows.2

